frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -208,5 +211,8 @@ async function checkCellTypes(page: Page, cellTypes = CELL_TYPE_FILTERS) {
  * Remove cell filter
  */
 async function removeCellFilter(page: Page) {
-  await page.getByTestId("CancelIcon").nth(0).click();
+  await await page
+    .getByTestId(`cell-type-tag-${CELL_TYPE_FILTERS[0]}`)
```

Comment:
quick questioN;

would this always try to find `CELL_TYPE_FILTERS[0]`?

For example, if we call `removeCellFilter()` twice, the second call would fail, since `CELL_TYPE_FILTERS[0]` is already removed?

Reply:
yes!